examples/graphviz_layout/plot_atlas.py

```diff
@@ -3,9 +3,10 @@
 Atlas
 =====
 
-Atlas of all graphs of 6 nodes or less.
+Atlas of all connected graphs with up to 6 nodes.
 
-This example needs Graphviz and PyGraphviz.
+This example uses Graphviz via PyGraphviz.
+It should show 142 graphs (oeis.org/A001349).
```

Comment:
It's probably not relevant to the example, but I was confused by the linked sequence: it looks like the value is 112 for `n=6`. I'm not sure that I'm interpreting this correctly though.

Reply:
Ah I think I can answer my own question - it's the sum of the terms up n=2 up to n=6.

Reply:
I also struggled with this so maybe a clarifying doc would be better.
We don't include the empty graph or the single node graph in the picture so it starts at n=2.
I'll add something...